templates/example_mixing_score.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>What's this stuff? What are the axes? What should the user be looking for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>